2016-04-10 22:45:07.263 [error] <0.98.0> gen_server mnesia2_recover terminated with reason: killed
2016-04-10 22:45:07.265 [error] <0.663.0> CRASH REPORT Process <0.663.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:07.263 [error] <0.98.0> CRASH REPORT Process mnesia2_recover with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:45:07.264 [error] <0.92.0> Supervisor mnesia2_sup had child mnesia2_kernel_sup started with mnesia2_kernel_sup:start() at <0.94.0> exit with reason killed in context child_terminated
2016-04-10 22:45:07.264 [error] <0.92.0> Supervisor mnesia2_sup had child mnesia2_kernel_sup started with mnesia2_kernel_sup:start() at <0.94.0> exit with reason reached_max_restart_intensity in context shutdown
2016-04-10 22:45:07.264 [error] <0.96.0> gen_server mnesia2_subscr terminated with reason: killed
2016-04-10 22:45:07.264 [error] <0.96.0> CRASH REPORT Process mnesia2_subscr with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:45:07.264 [error] <0.95.0> gen_server mnesia2_monitor terminated with reason: killed
2016-04-10 22:45:07.649 [error] <0.793.0> Mnesia2('test946@testing-worker-linux-docker-c74eb140-3365-linux-10'): ** ERROR ** (ignoring core) ** FATAL ** A fallback is installed and Mnesia2 must be restarted. Forcing shutdown after mnesia2_down from 'd1087130o1087126n1087126@testing-worker-linux-docker-c74eb140-3365-linux-10'...

2016-04-10 22:45:07.649 [error] <0.794.0> Supervisor mnesia2_kernel_sup had child mnesia2_locker started with mnesia2_locker:start() at <0.797.0> exit with reason killed in context child_terminated
2016-04-10 22:45:07.649 [error] <0.794.0> Supervisor mnesia2_kernel_sup had child mnesia2_locker started with mnesia2_locker:start() at <0.797.0> exit with reason reached_max_restart_intensity in context shutdown
2016-04-10 22:45:08.522 [error] <0.516.0> gen_server <0.516.0> terminated with reason: killed
2016-04-10 22:45:08.523 [error] <0.1088.0> gen_server <0.1088.0> terminated with reason: killed
2016-04-10 22:45:08.522 [error] <0.516.0> CRASH REPORT Process <0.516.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:45:08.524 [error] <0.1088.0> CRASH REPORT Process <0.1088.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:45:08.523 [error] <0.449.0> CRASH REPORT Process <0.449.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:08.524 [error] <0.1018.0> CRASH REPORT Process <0.1018.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:09.178 [error] <0.1318.0> gen_server <0.1318.0> terminated with reason: killed
2016-04-10 22:45:09.176 [error] <0.660.0> gen_server <0.660.0> terminated with reason: killed
2016-04-10 22:45:09.178 [error] <0.1318.0> CRASH REPORT Process <0.1318.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:45:09.176 [error] <0.660.0> CRASH REPORT Process <0.660.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:45:09.178 [error] <0.1253.0> CRASH REPORT Process <0.1253.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:09.176 [error] <0.538.0> CRASH REPORT Process <0.538.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:09.404 [error] <0.813.0> gen_server <0.813.0> terminated with reason: killed
2016-04-10 22:45:09.406 [error] <0.1485.0> CRASH REPORT Process <0.1485.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:09.404 [error] <0.813.0> CRASH REPORT Process <0.813.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:45:09.405 [error] <0.752.0> CRASH REPORT Process <0.752.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:10.215 [error] <0.1016.0> gen_server <0.1016.0> terminated with reason: killed
2016-04-10 22:45:10.220 [error] <0.1716.0> CRASH REPORT Process <0.1716.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:10.215 [error] <0.1016.0> CRASH REPORT Process <0.1016.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:45:10.216 [error] <0.953.0> CRASH REPORT Process <0.953.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:10.777 [error] <0.1842.0> CRASH REPORT Process <0.1842.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:11.061 [error] <0.1974.0> CRASH REPORT Process <0.1974.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:11.059 [error] <0.1222.0> gen_server <0.1222.0> terminated with reason: killed
2016-04-10 22:45:11.059 [error] <0.1222.0> CRASH REPORT Process <0.1222.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:45:11.059 [error] <0.1159.0> CRASH REPORT Process <0.1159.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:11.914 [error] <0.2173.0> gen_server <0.2173.0> terminated with reason: killed
2016-04-10 22:45:11.914 [error] <0.1322.0> gen_server <0.1322.0> terminated with reason: killed
2016-04-10 22:45:11.914 [error] <0.1322.0> CRASH REPORT Process <0.1322.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:45:11.914 [error] <0.2173.0> CRASH REPORT Process <0.2173.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:45:11.915 [error] <0.2107.0> CRASH REPORT Process <0.2107.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:12.244 [error] <0.1430.0> gen_server <0.1430.0> terminated with reason: killed
2016-04-10 22:45:12.246 [error] <0.2313.0> gen_server <0.2313.0> terminated with reason: killed
2016-04-10 22:45:12.245 [error] <0.1430.0> CRASH REPORT Process <0.1430.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:45:12.246 [error] <0.2313.0> CRASH REPORT Process <0.2313.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:45:12.245 [error] <0.1367.0> CRASH REPORT Process <0.1367.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:12.246 [error] <0.2248.0> CRASH REPORT Process <0.2248.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:13.216 [error] <0.2634.0> CRASH REPORT Process <0.2634.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:13.309 [error] <0.2762.0> CRASH REPORT Process <0.2762.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:15.893 [error] <0.2966.0> CRASH REPORT Process <0.2966.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:20.185 [error] <0.3240.0> CRASH REPORT Process <0.3240.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:22.344 [error] <0.3360.0> Mnesia2('test946@testing-worker-linux-docker-c74eb140-3365-linux-10'): Cannot report event {mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_3,shutdown]}}: {badarg,{mnesia2_event,{mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_3,shutdown]}}}} ({error,{already_started,<0.3408.0>}})

2016-04-10 22:45:22.344 [error] <0.3362.0> Mnesia2('test946@testing-worker-linux-docker-c74eb140-3365-linux-10'): Cannot report event {mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_5,shutdown]}}: {badarg,{mnesia2_event,{mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_5,shutdown]}}}} ({error,{already_started,<0.3408.0>}})

2016-04-10 22:45:22.344 [error] <0.3361.0> Mnesia2('test946@testing-worker-linux-docker-c74eb140-3365-linux-10'): Cannot report event {mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_4,shutdown]}}: {badarg,{mnesia2_event,{mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_4,shutdown]}}}} ({error,{already_started,<0.3408.0>}})

2016-04-10 22:45:22.344 [error] <0.3359.0> Mnesia2('test946@testing-worker-linux-docker-c74eb140-3365-linux-10'): Cannot report event {mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_2,shutdown]}}: {badarg,{mnesia2_event,{mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_2,shutdown]}}}} ({error,{already_started,<0.3408.0>}})

2016-04-10 22:45:22.344 [error] <0.3366.0> Mnesia2('test946@testing-worker-linux-docker-c74eb140-3365-linux-10'): Cannot report event {mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_7,shutdown]}}: {badarg,{mnesia2_event,{mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_7,shutdown]}}}} ({error,{already_started,<0.3408.0>}})

2016-04-10 22:45:22.344 [error] <0.3367.0> Mnesia2('test946@testing-worker-linux-docker-c74eb140-3365-linux-10'): Cannot report event {mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_8,shutdown]}}: {badarg,{mnesia2_event,{mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_8,shutdown]}}}} ({error,{already_started,<0.3408.0>}})

2016-04-10 22:45:22.345 [error] <0.3365.0> Mnesia2('test946@testing-worker-linux-docker-c74eb140-3365-linux-10'): Cannot report event {mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_6,shutdown]}}: {badarg,{mnesia2_event,{mnesia2_system_event,{mnesia2_info,"~p terminated: ~p~n",[mnesia2_tm_6,shutdown]}}}} ({error,{already_started,<0.3408.0>}})

2016-04-10 22:45:32.573 [error] <0.3693.0> CRASH REPORT Process <0.3693.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:34.670 [error] <0.3757.0> CRASH REPORT Process <0.3757.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:45:38.890 [error] <0.3892.0> Mnesia2('test946@testing-worker-linux-docker-c74eb140-3365-linux-10'): ** WARNING ** Mnesia2 is overloaded: {dump_log,write_threshold}

2016-04-10 22:45:52.552 [error] <0.3889.0> CRASH REPORT Process <0.3889.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:00.696 [error] <0.4471.0> CRASH REPORT Process <0.4471.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:06.265 [error] <0.4687.0> CRASH REPORT Process <0.4687.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:08.324 [error] <0.4754.0> CRASH REPORT Process <0.4754.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:11.442 [error] <0.1841.0> gen_server <0.1841.0> terminated with reason: killed
2016-04-10 22:46:11.442 [error] <0.1841.0> CRASH REPORT Process <0.1841.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:11.442 [error] <0.1780.0> CRASH REPORT Process <0.1780.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:14.504 [error] <0.4846.0> CRASH REPORT Process <0.4846.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:15.555 [error] <0.1848.0> CRASH REPORT Process <0.1848.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:16.663 [error] <0.1929.0> CRASH REPORT Process <0.1929.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:16.664 [error] <0.4952.0> CRASH REPORT Process <0.4952.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:33.506 [error] <0.2182.0> gen_server <0.2182.0> terminated with reason: killed
2016-04-10 22:46:33.506 [error] <0.2182.0> CRASH REPORT Process <0.2182.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:33.507 [error] <0.2121.0> CRASH REPORT Process <0.2121.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:33.527 [error] <0.5286.0> Supervisor mnesia2_kernel_sup had child mnesia2_checkpoint_sup started with mnesia2_checkpoint_sup:start() at <0.5303.0> exit with reason killed in context child_terminated
2016-04-10 22:46:33.527 [error] <0.5286.0> Supervisor mnesia2_kernel_sup had child mnesia2_checkpoint_sup started with mnesia2_checkpoint_sup:start() at <0.5303.0> exit with reason reached_max_restart_intensity in context shutdown
2016-04-10 22:46:33.529 [error] <0.5282.0> CRASH REPORT Process <0.5282.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:34.245 [error] <0.5450.0> gen_server <0.5450.0> terminated with reason: killed
2016-04-10 22:46:34.245 [error] <0.5450.0> CRASH REPORT Process <0.5450.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:34.246 [error] <0.5370.0> CRASH REPORT Process <0.5370.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:35.370 [error] <0.2209.0> CRASH REPORT Process <0.2209.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:36.464 [error] <0.5566.0> gen_server <0.5566.0> terminated with reason: killed
2016-04-10 22:46:36.464 [error] <0.2312.0> CRASH REPORT Process <0.2312.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:36.464 [error] <0.5566.0> CRASH REPORT Process <0.5566.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:36.465 [error] <0.5493.0> CRASH REPORT Process <0.5493.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:38.577 [error] <0.2408.0> gen_server <0.2408.0> terminated with reason: killed
2016-04-10 22:46:38.578 [error] <0.5677.0> gen_server <0.5677.0> terminated with reason: killed
2016-04-10 22:46:38.577 [error] <0.2408.0> CRASH REPORT Process <0.2408.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:38.578 [error] <0.5677.0> CRASH REPORT Process <0.5677.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:38.579 [error] <0.2387.0> CRASH REPORT Process <0.2387.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:38.582 [error] <0.5604.0> CRASH REPORT Process <0.5604.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:40.648 [error] <0.5735.0> gen_server <0.5735.0> terminated with reason: killed
2016-04-10 22:46:40.648 [error] <0.5735.0> CRASH REPORT Process <0.5735.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:40.649 [error] <0.5684.0> CRASH REPORT Process <0.5684.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:42.082 [error] <0.2446.0> CRASH REPORT Process <0.2446.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:43.594 [error] <0.5813.0> Supervisor mnesia2_kernel_sup had child mnesia2_checkpoint_sup started with mnesia2_checkpoint_sup:start() at <0.5819.0> exit with reason killed in context child_terminated
2016-04-10 22:46:43.594 [error] <0.2595.0> gen_server <0.2595.0> terminated with reason: killed
2016-04-10 22:46:43.594 [error] <0.5813.0> Supervisor mnesia2_kernel_sup had child mnesia2_checkpoint_sup started with mnesia2_checkpoint_sup:start() at <0.5819.0> exit with reason reached_max_restart_intensity in context shutdown
2016-04-10 22:46:43.594 [error] <0.2595.0> CRASH REPORT Process <0.2595.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:43.595 [error] <0.5811.0> Supervisor mnesia2_sup had child mnesia2_kernel_sup started with mnesia2_kernel_sup:start() at <0.5813.0> exit with reason shutdown in context child_terminated
2016-04-10 22:46:43.595 [error] <0.2535.0> CRASH REPORT Process <0.2535.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:43.595 [error] <0.5811.0> Supervisor mnesia2_sup had child mnesia2_kernel_sup started with mnesia2_kernel_sup:start() at <0.5813.0> exit with reason reached_max_restart_intensity in context shutdown
2016-04-10 22:46:45.723 [error] <0.2637.0> gen_server <0.2637.0> terminated with reason: killed
2016-04-10 22:46:45.736 [error] <0.5904.0> CRASH REPORT Process <0.5904.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:45.723 [error] <0.2637.0> CRASH REPORT Process <0.2637.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:45.724 [error] <0.2616.0> CRASH REPORT Process <0.2616.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:47.857 [error] <0.5998.0> CRASH REPORT Process <0.5998.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:47.857 [error] <0.2686.0> CRASH REPORT Process <0.2686.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:50.056 [error] <0.2761.0> CRASH REPORT Process <0.2761.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:51.083 [error] emulator Error in process <0.6153.0> on node 'test946@testing-worker-linux-docker-c74eb140-3365-linux-10' with exit value: {badarg,[{erlang,unlink,[undefined],[]},{mnesia2_schema,schema_coordinator,3,[{file,"src/mnesia2_schema.erl"},{line,619}]}]}


2016-04-10 22:46:51.084 [error] <0.6084.0> CRASH REPORT Process <0.6084.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:58.180 [error] <0.2907.0> gen_server <0.2907.0> terminated with reason: killed
2016-04-10 22:46:58.183 [error] <0.6215.0> gen_server <0.6215.0> terminated with reason: killed
2016-04-10 22:46:58.180 [error] <0.2907.0> CRASH REPORT Process <0.2907.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:58.183 [error] <0.6215.0> CRASH REPORT Process <0.6215.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:58.181 [error] <0.2831.0> CRASH REPORT Process <0.2831.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:58.183 [error] <0.6162.0> CRASH REPORT Process <0.6162.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:59.756 [error] <0.2996.0> gen_server <0.2996.0> terminated with reason: killed
2016-04-10 22:46:59.756 [error] <0.2996.0> CRASH REPORT Process <0.2996.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:59.756 [error] <0.3001.0> gen_server <0.3001.0> terminated with reason: killed
2016-04-10 22:46:59.756 [error] <0.3001.0> CRASH REPORT Process <0.3001.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:59.756 [error] <0.2930.0> CRASH REPORT Process <0.2930.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:59.839 [error] <0.6369.0> gen_server <0.6369.0> terminated with reason: killed
2016-04-10 22:46:59.839 [error] <0.6369.0> CRASH REPORT Process <0.6369.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:59.839 [error] <0.6346.0> gen_server <0.6346.0> terminated with reason: killed
2016-04-10 22:46:59.840 [error] <0.6346.0> CRASH REPORT Process <0.6346.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:59.840 [error] <0.6385.0> gen_server <0.6385.0> terminated with reason: killed
2016-04-10 22:46:59.840 [error] <0.6385.0> CRASH REPORT Process <0.6385.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:46:59.841 [error] <0.6272.0> CRASH REPORT Process <0.6272.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:46:59.862 [error] <0.276.0> gen_server <0.276.0> terminated with reason: killed
2016-04-10 22:46:59.862 [error] <0.276.0> CRASH REPORT Process <0.276.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:00.816 [error] <0.367.0> gen_server <0.367.0> terminated with reason: killed
2016-04-10 22:47:00.816 [error] <0.367.0> CRASH REPORT Process <0.367.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:00.816 [error] <0.358.0> gen_server <0.358.0> terminated with reason: killed
2016-04-10 22:47:00.816 [error] <0.358.0> CRASH REPORT Process <0.358.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:00.816 [error] <0.286.0> CRASH REPORT Process <0.286.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:03.275 [error] <0.6395.0> CRASH REPORT Process <0.6395.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:03.296 [error] <0.3219.0> gen_server <0.3219.0> terminated with reason: killed
2016-04-10 22:47:03.296 [error] <0.3219.0> CRASH REPORT Process <0.3219.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:03.298 [error] <0.3117.0> CRASH REPORT Process <0.3117.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:04.252 [error] <0.572.0> gen_server <0.572.0> terminated with reason: killed
2016-04-10 22:47:04.252 [error] <0.572.0> CRASH REPORT Process <0.572.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:04.252 [error] <0.566.0> gen_server <0.566.0> terminated with reason: killed
2016-04-10 22:47:04.252 [error] <0.566.0> CRASH REPORT Process <0.566.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:04.252 [error] <0.488.0> CRASH REPORT Process <0.488.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:04.421 [error] <0.6617.0> gen_server <0.6617.0> terminated with reason: killed
2016-04-10 22:47:04.421 [error] <0.6617.0> CRASH REPORT Process <0.6617.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:04.442 [error] <0.3326.0> gen_server <0.3326.0> terminated with reason: killed
2016-04-10 22:47:04.421 [error] <0.6612.0> gen_server <0.6612.0> terminated with reason: killed
2016-04-10 22:47:04.442 [error] <0.3326.0> CRASH REPORT Process <0.3326.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:04.421 [error] <0.6612.0> CRASH REPORT Process <0.6612.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:04.443 [error] <0.3231.0> CRASH REPORT Process <0.3231.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:04.421 [error] <0.6591.0> gen_server <0.6591.0> terminated with reason: killed
2016-04-10 22:47:04.421 [error] <0.6591.0> CRASH REPORT Process <0.6591.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:04.421 [error] <0.6512.0> CRASH REPORT Process <0.6512.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:05.376 [error] <0.3342.0> Supervisor mnesia2_kernel_sup had child mnesia2_checkpoint_sup started with mnesia2_checkpoint_sup:start() at <0.3367.0> exit with reason killed in context child_terminated
2016-04-10 22:47:05.377 [error] <0.3342.0> Supervisor mnesia2_kernel_sup had child mnesia2_checkpoint_sup started with mnesia2_checkpoint_sup:start() at <0.3367.0> exit with reason reached_max_restart_intensity in context shutdown
2016-04-10 22:47:05.377 [error] <0.3421.0> gen_server <0.3421.0> terminated with reason: killed
2016-04-10 22:47:05.377 [error] <0.3421.0> CRASH REPORT Process <0.3421.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:05.377 [error] <0.3413.0> gen_server <0.3413.0> terminated with reason: killed
2016-04-10 22:47:05.377 [error] <0.3413.0> CRASH REPORT Process <0.3413.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:05.378 [error] <0.3338.0> CRASH REPORT Process <0.3338.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:06.551 [error] <0.3487.0> gen_server <0.3487.0> terminated with reason: killed
2016-04-10 22:47:06.551 [error] <0.3487.0> CRASH REPORT Process <0.3487.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:06.551 [error] <0.3459.0> gen_server <0.3459.0> terminated with reason: killed
2016-04-10 22:47:06.551 [error] <0.3459.0> CRASH REPORT Process <0.3459.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:06.551 [error] <0.3503.0> gen_server <0.3503.0> terminated with reason: killed
2016-04-10 22:47:06.551 [error] <0.3503.0> CRASH REPORT Process <0.3503.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:06.552 [error] <0.3436.0> CRASH REPORT Process <0.3436.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:07.683 [error] <0.3514.0> CRASH REPORT Process <0.3514.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:07.683 [error] <0.691.0> Supervisor mnesia2_sup had child mnesia2_kernel_sup started with mnesia2_kernel_sup:start() at <0.693.0> exit with reason killed in context child_terminated
2016-04-10 22:47:07.686 [error] <0.6697.0> gen_server <0.6697.0> terminated with reason: killed
2016-04-10 22:47:07.683 [error] <0.691.0> Supervisor mnesia2_sup had child mnesia2_kernel_sup started with mnesia2_kernel_sup:start() at <0.693.0> exit with reason reached_max_restart_intensity in context shutdown
2016-04-10 22:47:07.686 [error] <0.6697.0> CRASH REPORT Process <0.6697.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:07.683 [error] <0.695.0> gen_server mnesia2_subscr terminated with reason: killed
2016-04-10 22:47:07.686 [error] <0.6746.0> gen_server <0.6746.0> terminated with reason: killed
2016-04-10 22:47:07.683 [error] <0.695.0> CRASH REPORT Process mnesia2_subscr with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:07.687 [error] <0.6746.0> CRASH REPORT Process <0.6746.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:07.683 [error] <0.694.0> gen_server mnesia2_monitor terminated with reason: killed
2016-04-10 22:47:07.687 [error] <0.6625.0> CRASH REPORT Process <0.6625.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:07.683 [error] <0.697.0> gen_server mnesia2_recover terminated with reason: killed
2016-04-10 22:47:07.683 [error] <0.697.0> CRASH REPORT Process mnesia2_recover with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:07.683 [error] <0.714.0> gen_server mnesia2_snmp_sup terminated with reason: killed
2016-04-10 22:47:07.683 [error] <0.714.0> CRASH REPORT Process mnesia2_snmp_sup with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:07.886 [error] <0.3656.0> gen_server <0.3656.0> terminated with reason: killed
2016-04-10 22:47:07.886 [error] <0.3656.0> CRASH REPORT Process <0.3656.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:07.886 [error] <0.3593.0> CRASH REPORT Process <0.3593.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:08.065 [error] <0.3851.0> gen_server <0.3851.0> terminated with reason: killed
2016-04-10 22:47:08.066 [error] <0.3851.0> CRASH REPORT Process <0.3851.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:08.067 [error] <0.3778.0> CRASH REPORT Process <0.3778.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:09.323 [error] <0.7039.0> gen_server <0.7039.0> terminated with reason: killed
2016-04-10 22:47:09.324 [error] <0.7039.0> CRASH REPORT Process <0.7039.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:09.325 [error] <0.4020.0> gen_server <0.4020.0> terminated with reason: killed
2016-04-10 22:47:09.324 [error] <0.6965.0> CRASH REPORT Process <0.6965.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:09.325 [error] <0.4020.0> CRASH REPORT Process <0.4020.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:09.325 [error] <0.3960.0> CRASH REPORT Process <0.3960.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:09.813 [error] <0.911.0> gen_server <0.911.0> terminated with reason: killed
2016-04-10 22:47:09.813 [error] <0.7072.0> gen_server mnesia2_subscr terminated with reason: killed
2016-04-10 22:47:09.813 [error] <0.911.0> CRASH REPORT Process <0.911.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:09.813 [error] <0.7072.0> CRASH REPORT Process mnesia2_subscr with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:09.813 [error] <0.904.0> gen_server <0.904.0> terminated with reason: killed
2016-04-10 22:47:09.813 [error] <0.904.0> CRASH REPORT Process <0.904.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:09.813 [error] <0.840.0> CRASH REPORT Process <0.840.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:19.974 [error] <0.7272.0> Supervisor mnesia2_sup had child mnesia2_kernel_sup started with mnesia2_kernel_sup:start() at undefined exit with reason killed in context start_error
2016-04-10 22:47:19.974 [error] <0.7275.0> gen_server mnesia2_monitor terminated with reason: killed
2016-04-10 22:47:19.974 [error] <0.7278.0> gen_server mnesia2_recover terminated with reason: killed
2016-04-10 22:47:19.974 [error] <0.7278.0> CRASH REPORT Process mnesia2_recover with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:19.974 [error] <0.7298.0> gen_server mnesia2_snmp_sup terminated with reason: killed
2016-04-10 22:47:19.975 [error] <0.7298.0> CRASH REPORT Process mnesia2_snmp_sup with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:19.975 [error] <0.7270.0> CRASH REPORT Process <0.7270.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,mnesia2_kernel_sup,killed}},{mnesia2_sup,start,[normal,[]]}} in application_master:init/4 line 133
2016-04-10 22:47:19.975 [error] <0.7275.0> CRASH REPORT Process mnesia2_monitor with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:19.977 [error] <0.7276.0> gen_server mnesia2_subscr terminated with reason: {aborted,{node_not_running,'test946@testing-worker-linux-docker-c74eb140-3365-linux-10'}} in mnesia2:abort/1 line 313
2016-04-10 22:47:19.977 [error] <0.7276.0> CRASH REPORT Process mnesia2_subscr with 0 neighbours exited with reason: {aborted,{node_not_running,'test946@testing-worker-linux-docker-c74eb140-3365-linux-10'}} in gen_server:terminate/7 line 792
2016-04-10 22:47:20.058 [error] <0.4279.0> gen_server <0.4279.0> terminated with reason: killed
2016-04-10 22:47:20.058 [error] <0.1081.0> gen_server <0.1081.0> terminated with reason: killed
2016-04-10 22:47:20.059 [error] <0.4279.0> CRASH REPORT Process <0.4279.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:20.058 [error] <0.1081.0> CRASH REPORT Process <0.1081.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:20.059 [error] <0.4226.0> CRASH REPORT Process <0.4226.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
2016-04-10 22:47:20.059 [error] <0.1075.0> gen_server <0.1075.0> terminated with reason: killed
2016-04-10 22:47:20.059 [error] <0.1075.0> CRASH REPORT Process <0.1075.0> with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2016-04-10 22:47:20.059 [error] <0.1022.0> CRASH REPORT Process <0.1022.0> with 0 neighbours exited with reason: killed in application_master:terminate/2 line 227
